### DISCUSSION TITLE
Contributing to flet

### DISCUSSION THREAD

**Discussion by clstaudt** [flet-dev/flet#548] [Q&A]
On the one hand flet development is moving forward quickly and the community is active and responsive. On the other there is still so much functionality from the Flutter and Python world that is not yet available in flet. This includes basic components that may be within reach for new contributors.

What is the project's status in terms of the ability to onboard contributors and review/accept contributions?

**Comment by FeodorFitsner** [comment]
We'd love to see contributors at this point! I believe the framework is ready for contributions.
If you are thinking to start working on something (like [DatePicker](https://github.com/flet-dev/flet/issues/86#issuecomment-1303785568)?) create a new issue or use that one, so we can discuss in details the course of action, what should be done to implement a new control, where and how. Let me know.

**Reply by ndonkoHenri** [reply]
Little question,
Only python knowledge is required?
In other words, must we know how to code in Dart/Flutter ?

Or briefly, what is required (tech stack knowledge) to get started  creating controls?

**Reply by FeodorFitsner** [reply]
It depends on the control you are going to build.

If it's a "composite" control made of other existing Flet controls then Python knowledge is only required.

If it's a new control implementing a new Flutter widget then both Python and Flutter/Dart knowledge are required.
